client/rhel/spacewalk-client-tools/src/firstboot-legacy-rhel6/rhn_login_gui.py

```diff
@@ -35,8 +35,8 @@ class moduleClass(Module):
     def __init__(self):
         Module.__init__(self)
         self.priority = 108.5
-        self.sidebarTitle = _("Red Hat Account")
-        self.title = _("Red Hat Account")
+        self.sidebarTitle = _("Uyuni Account")
+        self.title = _("Uyuni Account")
```

Comment:
Same. It seems this was for users logging in to RHN. That would be SCC in SUSE, or nothing in Uyuni. But probably we don't use this codepath anymore :-?

Reply:
I don't think that the client allowed direct connection to RHN and I think that this is the same as the other cases above, when this really meant Satellite, not RHN.